src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsCreator.php

```diff
@@ -83,77 +83,72 @@ public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredential
      *
      * @return \Generated\Shared\Transfer\UnzerCredentialsResponseTransfer
      */
-    public function createUnzerCredentialsAndSetUnzerNotificationUrl(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer
```

Comment:
I'm not sure that it's expected refactoring because as I understood methods `createUnzerCredentials()` and `createUnzerCredentialsAndSetUnzerNotificationUrl` contained different logic.
@kkicheglovspryker Could you check it?

Reply:
All is fine, we agreed that for now "create credentials w/o register webhook" is not required